python-malformed-yaml/github_service.py

```diff
@@ -0,0 +1,20 @@
+from github import Github
+
+
+class GitHubService():
```

Comment:
Are there any tests for this class?

Reply:
er, no...

Reply:
...not yet....

Reply:
Actually I don't think there's really any logic to test - the GitHubService methods just collect specific things directly using `github`, could test that the type it returns is as expected 🤷‍♀️